### PR TITLE
Draft README

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY
Draft README

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
...
+```
+
+
+### See Also:
+
+* [Amazon Web Services Guide](https://docs.ansible.com/ansible/latest/scenario_guides/guide_aws.html)
+* [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+
+## Contributing to this collection
+
```
Comment (wimnat) [inline README.md:52]: I think this section should link to CONTRIBUTING.md with contributing specifics documented in that file.

**Comment by jillr** [comment]
@tremble @wimnat Creating a CONTRIBUTING.md is definitely still an outstanding todo, I'd like to do that under a separate PRs so this one can land though.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+    ansible-galaxy collection install community.aws
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: community.aws
+    version: 0.1.1
```
[the remark this answers is not in the document]
Comment (jillr) [reply README.md:23]: What would you use instead?

[the remark this answers is not in the document]
Comment (tremble) [reply README.md:23]: I think I'd rather we either have two examples (one with versioning, one without), or just a version without.  'version: 0.1.1' will only be the right thing to copy and paste for a very limited time.

**Comment by tremble** [comment]
> I'd like to do that under a separate PRs so this one can land though.

Works for me.  I think at the very least this is a good enough start to be worth merging as is.  (In the spirit of not letting perfect be the enemy of good)

**Comment by wimnat** [comment]
> @tremble @wimnat Creating a CONTRIBUTING.md is definitely still an outstanding todo, I'd like to do that under a separate PRs so this one can land though.

Yeh I am happy to do separately but I think it's really important it is tackled soon.  Even a simple one that says "This is where and when we meet" so that people can then meet to discuss exactly what detail should go in there long term.

**Comment by jillr** [comment]
@wimnat Yep, we have a list of tasks we're working towards with all of the collections we manage that we're targeting for the next month or so.  Contrib guides for both amazon.aws and this collection will be a part of that.

**Comment by wimnat** [comment]
> @wimnat Yep, we have a list of tasks we're working towards with all of the collections we manage that we're targeting for the next month or so. Contrib guides for both amazon.aws and this collection will be a part of that.

Is there an Issue in ansible core that is tracking this that I can follow please?

**Comment by jillr** [comment]
@wimnat Hehe, we welcome your contributions to modules in amazon.aws too!  Things getting promoted to amazon.aws/core-supported just means they're in really great shape so thanks for that. :grin:

There's high level issues in https://github.com/ansible-collections/overview/issues but not a specific issue tracking any one given collection.  We're replicating a lot of this initial collection housekeeping across 20+ collections (cloud, networking, security, others) for this initial push so it's being tracked by Red Hat project management.  It's a tradeoff between transparency and having to duplicate and keep on top of the project management and tracking effort across dozens of repos - none of this is intended to be secret, it's just a logistics thing.  The list we hope to have done by mid June looks like this (most of these are done for both AWS collections now):

README in repo
TAGS in galaxy.yml
All plugin examples updated to FQCN
All plugin short descriptions updated if necessary 
CI in place for tagging and releasing to galaxy
ansible-test sanity enabled
ansible-test units enabled
ansible-test integration enabled
Code Coverage
Tested and green against 2.9
Tested and green against 2.10
Platform Guide Update (if applicable)
Contribution Guide in repo
Changelog generation in place
Update version_added to 1.0.0 in ALL modules doc string
Remove ALL ANSIBLE_METADATA in each module
Add table of plugins to README and rst for each in docs folder
Add runtime.yml along with short name redirection if applicable
Only license_file is specified in galaxy.yml (no `license` key)
Published to Galaxy
Added to ACD Build

**Comment by wimnat** [comment]
@jillr is there a kind of grand handover planned to an appointed community maintainers team for this collection (and others) or are things kind of continuing the same as 2.9 as far as PR approvals and merging goes at least for 2.10 ships and then make changes after that?
Sorry for all the questions!

**Comment by jillr** [comment]
@wimnat No worries, ask away!  The Ansible Community team (@gundalow et al) are already managing several of the community collections, notably `community.general`.  For some of the community collections that need a bit more domain expertise they will continue to be overseen more directly by the relevant teams within Ansible.  The cloud team will stay involved with `community.aws` and `community.vmware`.  We have the ability to promote members of the community as maintainers for a given collection just like we did in ansible/ansible with shipit and supershipit, and the communities for a collection also have the option now to define their own processes for that.

You mentioned meetings, I asked around in the `#ansible-aws` channel a couple months back, earlier in the collections process, and folks indicated they didn't see any need for a meeting.  I'd be happy to revisit this at any time though.

**Comment by jillr** [comment]
@tremble @wimnat I believe there are no outstanding change requests on this PR?  I've started drafting the contrib guide at https://github.com/ansible-collections/amazon.aws/pull/62, if y'all have no objections I'd like to merge this one.

**Comment by tremble** [comment]
Due to a mistake this PR is now reduced to some minor whitespace cleanup.  +1 to getting it merged and closing the PR.

**Comment by wimnat** [comment]
lgtm

**Pull request merged**